cardinal/server/server.go

```diff
@@ -56,16 +49,26 @@ func New(engine *ecs.Engine, wsEventHandler func(conn *websocket.Conn), opts ...
 	// Enable CORS
 	app.Use(cors.New())
 	setupRoutes(app, engine, wsEventHandler, s.config)
+	//	@title			Cardinal
+	//	@description	Backend server for World Engine
+	//	@version		0.0.1
+	//	@schemes		http ws
+	//	@BasePath		/
+	//	@consumes		application/json
+	//	@produces		application/json
+	setupRoutes(app, engine, wsEventHandler, s.config)
```

Comment:
duplicated setupRoutes call (L51, L59)

is there another place these comments can go? feels hidden here